app/src/main/java/com/android/sample/ui/dialogs/AddImageDialog.kt

```diff
@@ -100,12 +107,31 @@ fun AddImageDialog(onDismiss: () -> Unit, onGalleryClick: () -> Unit, onCameraCl
                     contentDescription = "Choose from gallery",
                     tint = Color.Gray)
               }
+
+          if (default) {
+            TextButton(
+                onClick = onSelectDefault,
+                modifier =
+                    Modifier.fillMaxWidth()
+                        .padding(STANDARD_PADDING.dp)
+                        .testTag("defaultImageButton")
+                        .border(
+                            width = BORDER_STROKE_SM.dp,
+                            color = Color.Cyan,
+                            shape = RoundedCornerShape(ROUNDED_CORNER_SHAPE_DEFAULT.dp))) {
+                  Text(
+                      text = "Select default picture",
+                      style = TextStyle(color = Color.Black, fontSize = SUBTITLE_FONTSIZE.sp))
+                  Spacer(modifier = Modifier.width(STANDARD_PADDING.dp))
+                  Icon(Icons.Filled.AddToPhotos, contentDescription = "Default", tint = Color.Gray)
+                }
+          }
         }
       }
 }
 
 @Preview
 @Composable
 fun AddImageDialogPreview() {
```

Comment:
This preview was only for displaying the composable right? Maybe it is better to remove it